samples/settings.gradle

```diff
@@ -0,0 +1,17 @@
+pluginManagement {
+	repositories {
+		maven { url 'https://repo.spring.io/milestone' }
+		maven { url 'https://repo.spring.io/snapshot' }
+		gradlePluginPortal()
+	}
+}
+
+include 'grpc-reactive'
```

Comment:
> [!NOTE]
> The `grpc-oauth2` has some Maven libs for testing and I just left out for now (we can add later)

Reply:
It’s just a test. There’s a new release of testjars that bundles them all together too, so we drop them soon.